crates/networking/manager/src/validate.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::{Ok, anyhow};
+use ream_beacon_chain::beacon_chain::BeaconChain;
+use ream_consensus::{
+    constants::MAX_BLOBS_PER_BLOCK_ELECTRA, electra::beacon_block::SignedBeaconBlock,
+    misc::compute_start_slot_at_epoch,
+};
+use ream_storage::tables::{Field, Table};
+
+#[derive(Debug)]
+pub enum ValidationResult {
+    Accept,
+    Ignore,
+    Reject,
+}
+
+pub async fn validate_beacon_block(
+    beacon_chain: &BeaconChain,
+    block: &SignedBeaconBlock,
+) -> anyhow::Result<ValidationResult> {
```

Comment:
if we are defining ValidationResult, shouldn't we have Error as a part of the result, instead of having to unwrap 2 results